src/mono/wasm/runtime/types.ts

```diff
@@ -181,6 +181,7 @@ export type DotnetModuleConfig = {
 
     config?: MonoConfig | MonoConfigError,
     configSrc?: string,
+
```

Comment:
If it's easy to drop this file out of the PR it'd be overall goodness, since it's whitespace-only